src/cli/index.js

```diff
@@ -47,6 +47,11 @@ async function runTests (argParser) {
     log.showSpinner();
 
     var testCafe       = await createTestCafe(opts.hostname, port1, port2);
+    var concurrency    = argParser.concurrency || 1;
+
+    if (argParser.remoteCount && argParser.remoteCount % concurrency)
+        throw new GeneralError(MESSAGE.invalidRemotesCount);
```

Comment:
Error message name should be more specific and show that it's related to concurrency option.